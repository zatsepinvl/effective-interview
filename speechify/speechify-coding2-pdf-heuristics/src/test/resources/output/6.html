            <html><body>
            <div style="display: flex; flex-direction: row">
            <div style="width: 50%">
            <h1>Expected</h1>
            <hr />
            <p><i>Action Research – Applied, Intervention, Collaborative, Practitioner, or Praxis Research?</i></p>
<p><i>13</i></p>
<p>reflections on the Aristotelian concept of praxis. Hence, my text is a call for more distinctions and thereby more distinction in our thinking.</p>
<p>It is my claim, then, that it is both possible and desirable to do action research as praxis-research in ways that transcend “intervening”, “collaborating”, “interacting”, and “applying” mainstream research methods and scientific theory, as a dialectical “Aufhebung” of these terms and practices. A dialectical “Aufhebung” of something is not the same as simply leaving the transcended something behind, deconstructing, cancelling, and abandoning it. Aufhebung means transcending, retaining, transforming, and improving at the same time, literally; to raise something to a new level, mainly by recontextualising it. This can be done by rethinking these terms within a comprehended praxis-research framework.</p>
<p>Several forms of what is often termed “practitioner”, “native”, or “insider” research: where the tasks of knowledge generation and research are done by people from within the different vocations, professions, organizations, and practical pursuits themselves, seem to be on the trail of praxis-research, but often with insufficient philosophical reflection and sometimes apparently reducing the task of action research to problem solving and improving practice, without explicit and clear theoretical ambitions (McNiff & Whitehead, 2011). Praxis-research may require practitioner-research. But, still, not all forms of practitioner research qualify as praxis-research, as when practitioners merely apply conventional research methods (observing the behaviour of others, reading documents, and asking questions). Practitioner-research may be necessary but still not sufficient for praxis-research. As I have tried to clarify elsewhere, even within what might be called informed Aristotelian praxis-research there are important differences which, although there is no space for it here, need to be addressed.4 My presentation here may</p>
<p>4 My own reconstruction of Aristotle differs from the most current “applied” way of presenting Aristotle on knowing by separating epistêmê from tékhnê and phrónêsis (cf. Flyvbjerg, 2001; Toulmin, 1996a, 1996b, 1996c, 2001; Schwandt, 2002; Ramírez, 1995; Dunne, 1993; Polkinghorne, 2004). The separation is usually done in order to emphasize phrónêsis as an independent alternative to epistêmê and tékhnê, or to “science” and “technology”. Phrónêsis is seen as deliberation connected to praxis, interpreted as approximately our everyday activities, contrasted to science and technique. Often, rhetoric is also mustered as the deliberative rationality most appropriate for</p>
            </div>
            <div style="width: 50%">
            <h1>Actual</h1>
            <hr />
            
            </div>
            </div>
            </body></html>