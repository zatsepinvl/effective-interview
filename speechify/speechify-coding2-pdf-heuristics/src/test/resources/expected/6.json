[
  {
    "type": "Marginalia",
    "text": "Action Research \u2013 Applied, Intervention, Collaborative, Practitioner, or Praxis Research?"
  },
  { "type": "Marginalia", "text": "13" },
  {
    "type": "Paragraph",
    "text": "reflections on the Aristotelian concept of praxis. Hence, my text is a call for more distinctions and thereby more distinction in our thinking."
  },
  {
    "type": "Paragraph",
    "text": "It is my claim, then, that it is both possible and desirable to do action research as praxis-research in ways that transcend \u201cintervening\u201d, \u201ccollaborating\u201d, \u201cinteracting\u201d, and \u201capplying\u201d mainstream research methods and scientific theory, as a dialectical \u201cAufhebung\u201d of these terms and practices. A dialectical \u201cAufhebung\u201d of something is not the same as simply leaving the transcended something behind, deconstructing, cancelling, and abandoning it. Aufhebung means transcending, retaining, transforming, and improving at the same time, literally; to raise something to a new level, mainly by recontextualising it. This can be done by rethinking these terms within a comprehended praxis-research framework."
  },
  {
    "type": "Paragraph",
    "text": "Several forms of what is often termed \u201cpractitioner\u201d, \u201cnative\u201d, or \u201cinsider\u201d research: where the tasks of knowledge generation and research are done by people from within the different vocations, professions, organizations, and practical pursuits themselves, seem to be on the trail of praxis-research, but often with insufficient philosophical reflection and sometimes apparently reducing the task of action research to problem solving and improving practice, without explicit and clear theoretical ambitions (McNiff & Whitehead, 2011). Praxis-research may require practitioner-research. But, still, not all forms of practitioner research qualify as praxis-research, as when practitioners merely apply conventional research methods (observing the behaviour of others, reading documents, and asking questions). Practitioner-research may be necessary but still not sufficient for praxis-research. As I have tried to clarify elsewhere, even within what might be called informed Aristotelian praxis-research there are important differences which, although there is no space for it here, need to be addressed.4 My presentation here may"
  },
  {
    "type": "Paragraph",
    "text": "4 My own reconstruction of Aristotle differs from the most current \u201capplied\u201d way of presenting Aristotle on knowing by separating epist\u00eam\u00ea from t\u00e9khn\u00ea and phr\u00f3n\u00easis (cf. Flyvbjerg, 2001; Toulmin, 1996a, 1996b, 1996c, 2001; Schwandt, 2002; Ram\u00edrez, 1995; Dunne, 1993; Polkinghorne, 2004). The separation is usually done in order to emphasize phr\u00f3n\u00easis as an independent alternative to epist\u00eam\u00ea and t\u00e9khn\u00ea, or to \u201cscience\u201d and \u201ctechnology\u201d. Phr\u00f3n\u00easis is seen as deliberation connected to praxis, interpreted as approximately our everyday activities, contrasted to science and technique. Often, rhetoric is also mustered as the deliberative rationality most appropriate for"
  }
]
